src/Type/PHPUnit/CreateMockDynamicReturnTypeExtension.php

```diff
@@ -41,12 +42,14 @@ public function getTypeFromMethodCall(MethodReflection $methodReflection, Method
 		}
 
 		$argType = $scope->getType($methodCall->args[$argumentIndex]->value);
-		if (!($argType instanceof TypeWithClassName)) {
+		if ($argType instanceof TypeWithClassName) {
+			$class = $argType->getClassName();
+		} elseif ($argType instanceof ConstantStringType) {
+			$class = $argType->getValue();
```

Comment:
You didn't add any tests for this.

Reply:
There were no tests for this before, that's why. I'll write some asap.